server/channels/api4/channel.go

```diff
@@ -24,6 +24,7 @@ func (api *API) InitChannel() {
 	api.BaseRoutes.Channels.Handle("/group/search", api.APISessionRequiredDisableWhenBusy(searchGroupChannels)).Methods("POST")
 	api.BaseRoutes.Channels.Handle("/group", api.APISessionRequired(createGroupChannel)).Methods("POST")
 	api.BaseRoutes.Channels.Handle("/members/{user_id:[A-Za-z0-9]+}/view", api.APISessionRequired(viewChannel)).Methods("POST")
+	api.BaseRoutes.Channels.Handle("/members/{user_id:[A-Za-z0-9]+}/view_many", api.APISessionRequired(viewMultipleChannels)).Methods("POST")
```

Comment:
It looks like that a difference from `viewChannel` is that we also call `SetActiveChannel` in that case. That means we aren't really "viewing" multiple channels. Might be better to just call it `mark_read` for the REST API, and `readMultipleChannels` as the function name.